src/test/java/org/humancellatlas/ingest/project/ProjectServiceTest.java

```diff
@@ -287,4 +293,133 @@ void givenSuggestionCreatesProjectSuccessfully() {
             assertThat(result).isEqualTo(persistentProject);
         }
     }
+
+
+    @Nested
+    class FilterProject {
+        private String primaryWranglerKey = "primaryWrangler";
+        private String isUpdateKey = "isUpdate";
+        private String wranglingStateKey = "wranglingState";
+        private String projectTitleKey = "content.project_core.project_title";
+        private String projectDescripionKey = "content.project_core.project_description";
+        private String projectShortNameKey = "content.project_core.project_short_name";
+
+        private String searchText = "search text";
+        private String wranglingStateValue = "NEW";
+        private String primaryWranglerValue = "primary wrangler 1";
+
+        Criteria getDefaultFilter() {
+            return Criteria.where(isUpdateKey).is(false);
+        }
+
+        Criteria filterByWrangler(String wrangler) {
+            return Criteria.where(primaryWranglerKey).is(wrangler);
+        }
+
+        Criteria filterByWranglingState(String wranglingState) {
+            return Criteria.where(wranglingStateKey).is(wranglingState);
+        }
+
+        Criteria filterBySearch(String search) {
+            return new Criteria().orOperator(
+                    Criteria.where(projectTitleKey).regex(search, "i"),
+                    Criteria.where(projectDescripionKey).regex(search, "i"),
+                    Criteria.where(projectShortNameKey).regex(search, "i")
+            );
+        }
+
+        Query buildQuery(ArrayList<Criteria> criteria_list) {
+            return new Query().addCriteria(new Criteria().andOperator(criteria_list.toArray
+                    (new Criteria[criteria_list.size()])));
+        }
+
+        @Test
+        @DisplayName("get all projects")
+        void getProjectsWithNoFilter() {
+            // given empty search filter
+            SearchFilter searchFilter = new SearchFilter();
+            Query actualResult = projectService.buildProjectsQuery(searchFilter);
+
+            Query expectedResult = this.buildQuery(new ArrayList<>(Arrays.asList(getDefaultFilter())));
+            assertEquals(expectedResult, actualResult);
```

Comment:
Please use assertj assertions.
They make the test code as well as the error messages a lot readable.
You can see an example in the test I added.